website/src/routes/api/(actions)/toBigint/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: toBigint
+description: Creates a to bigint transformation action.
+source: /actions/toBigint/toBigint.ts
+contributors:
+  - EskiMojo14
+---
+
+import { ApiList, Property } from '~/components';
+import { properties } from './properties';
+
+# toBigint
+
+Creates a to bigint transformation action.
+
+```ts
+const Action = v.toBigint<TInput, TMessage>(message);
+```
+
+## Generics
+
+- `TInput` <Property {...properties.TInput} />
+- `TMessage` <Property {...properties.TMessage} />
+
+## Parameters
+
+- `message` <Property {...properties.message} />
+
+### Explanation
+
+With `toBigint` you can transform the input to a bigint. If the input cannot be transformed, you can use `message` to customize the error message.
```

Comment:
This says the user can pass a `message` but the code at `library/src/actions/toBigint/toBigint.ts` does not actually accept that argument.

Reply:
good catch, added message argument to toBigint